utilities/inference_utils.py

```diff
@@ -438,6 +438,8 @@ def run_inference(self, cmd: str) -> str:
         if not res:
             raise ValueError(f"Inference failed with error: {err}\nOutput: {out}\nCommand: {cmd}")
 
+        LOGGER.info(f"Inference output: {out}")
```

Comment:
i would suggest to add  new line before and after o/p  for clean logs

Reply:
deon